config/samples/deployment-valuefrom.yaml

```diff
@@ -17,14 +17,21 @@ spec:
       containers:
         - name: doppler-test
           image: alpine
-          command: ['/bin/sh', '-c', 'apk add --no-cache tini > /dev/null 2>&1 && printenv | grep -v KUBERNETES_ && tini -s tail -f /dev/null'] # Print all non-Kubernetes environment variables
+          command:
+            - /bin/sh
+            - -c
+            # Print all non-Kubernetes environment variables
+            - apk add --no-cache tini > /dev/null 2>&1 &&
+              echo "### This is a simple deployment running with this env:" &&
+              printenv | grep -v KUBERNETES_ &&
+              tini -s tail -f /dev/null
           imagePullPolicy: Always
           env:
-            - name: MY_APP_SECRET # The name of the environment variable exposed in the container
+            - name: DOPPLER_CONFIG # The name of the environment variable exposed in the container
```

Comment:
Is this a correct change? If so can we rename it to DOPPLER_CONFIG_SECRETS? We use DOPPLER_CONFIG for the config slug usually.

Reply:
Yep! In this example, I’m pulling one secret from our Doppler config to set as an environment variable in our deployment. I changed it to DOPPLER_CONFIG because that’s guaranteed to be a valid secret name